model_prices_and_context_window.json

```diff
@@ -6086,6 +5959,30 @@
         "litellm_provider": "together_ai",
         "mode": "embedding"
     },
+    "together_ai/meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo": {
+        "input_cost_per_token": 0.00000018,
+        "output_cost_per_token": 0.00000018,
+        "litellm_provider": "together_ai",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "mode": "chat"
+    },
+    "together_ai/meta-llama/Meta-Llama-3.1-70B-Instruct-Turbo": {
+        "input_cost_per_token": 0.00000088,
+        "output_cost_per_token": 0.00000088,
+        "litellm_provider": "together_ai",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "mode": "chat"
+    },
+    "together_ai/meta-llama/Meta-Llama-3.1-405B-Instruct-Turbo": {
+        "input_cost_per_token": 0.0000035,
+        "output_cost_per_token": 0.0000035,
+        "litellm_provider": "together_ai",
+        "supports_function_calling": true,
+        "supports_parallel_function_calling": true,
+        "mode": "chat"
+    },
```

Comment:
@krrishdholakia can you explain why we don't also add supports_function_calling for mistralai/Mixtral-8x7B-Instruct-v0.1 and mistralai/Mistral-7B-Instruct-v0.1 for together AI? https://docs.together.ai/docs/function-calling

and why we don't add supports_response_schema = True for  the models in https://docs.together.ai/docs/json-mode, and default to False for others?

Reply:
> we don't also add supports_function_calling for mistralai/Mixtral-8x7B-Instruct-v0.1 and mistralai/Mistral-7B-Instruct-v0.1 for

@adam-simple those already exist - https://github.com/BerriAI/litellm/blob/fb1ec4b6be70ada08475a9f52fef2af31c5f6c53/model_prices_and_context_window.json#L6130

>  we don't add supports_response_schema = True for the models in https://docs.together.ai/docs/json-mode

good point, i'll do that in today's pr

Reply:
Thanks! It seems we should also modify the together_ai/chat get_supported_openai_params to reference this field similar to how we do for supports_function_calling, or am I mistaken?

Reply:
no - because as long as the model supports function calling we can support json schema input to the model

Reply:
then does together have the wrong documentation? meta-llama/Meta-Llama-3.1-405B-Instruct-Turbo is available for function calling but not for json mode

https://docs.together.ai/docs/function-calling

https://docs.together.ai/docs/json-mode

Reply:
@krrishdholakia 

meta-llama/Meta-Llama-3.1-405B-Instruct-Turbo supports function calling but does not support json mode. So the code as it's written will see that it supports function calling, so it does not remove the response_format from the optional params.

This will cause an error from Together, no?

Reply:
> meta-llama/Meta-Llama-3.1-405B-Instruct-Turbo

i couldn't get a call working successfully to confirm this. I do believe it supports json mode as well since all tgai models (incl. the same llama 3.1 models but lower param size support it)

Reply:
```
curl -X POST "https://api.together.xyz/v1/chat/completions" \
  -H "Authorization: Bearer $TOGETHER_API_KEY" \
  -H "Content-Type: application/json" \
  -d '{
    "model": "meta-llama/Meta-Llama-3.1-405B-Instruct-Turbo",
    "messages": [],
    "max_tokens": 512,
    "temperature": 0.7,
    "top_p": 0.7,
    "top_k": 50,
    "repetition_penalty": 1,
    "stop": ["<|eot_id|>","<|eom_id|>"],
    "stream": true,
    "response_format": {
        "type": "json_object"
    }
  }'
```
run the above in your terminal after setting TOGETHER_API_KEY to be your together api key, and you will get:

`{
  "error": {
    "message": "meta-llama/Meta-Llama-3.1-405B-Instruct-Turbo is not supported for JSON mode/function calling",
    "type": "invalid_request_error",
    "param": null,
    "code": "constraints_model"
  }
}`

Reply:
@krrishdholakia ?